.taskcluster.yml

```diff
@@ -41,7 +41,7 @@ tasks:
               $switch:
                   'tasks_for[:19] == "github-pull-request"': ${event.pull_request.base.ref}
                   'tasks_for == "github-push" && event.base_ref': ${event.base_ref}
-                  'tasks_for == "github-push"': ${event.ref}
+                  'tasks_for == "github-push" && !event.base_ref': ${event.ref}
```

Comment:
I'm struggling to understand how this comes into play for release events. Isn't `tasks_for` set to `github-release` for those cases?

Reply:
Oh, I bet it's not the release event, but the tag push when you create a tag along with the release. I'll update the commit message.

Reply:
See, e.g:
https://github.com/mozilla-releng/mozilla-taskgraph/commit/a84b60cc431babd3df1e01f2d8740ba9668e0ec0

I think it's harmless in this case because the tag push wouldn't render a Decision task anyway, but might as well get rid of those comments?

Reply:
Oh wait, I don't think the tag push would get filtered away? In which case we'd be running a second Decision task against that revision.. Not sure if that's a problem worth fixing or not though :/